mypy_django_plugin/django/context.py

```diff
@@ -198,7 +198,12 @@ def get_expected_types(self, api: TypeChecker, model_cls: Type[Model], *, method
                 # it's generic, so cannot set specific model
                 field_name = field.name
                 gfk_info = helpers.lookup_class_typeinfo(api, field.__class__)
-                gfk_set_type = helpers.get_private_descriptor_type(gfk_info, "_pyi_private_set_type", is_nullable=True)
+                if gfk_info is None:
```

Comment:
Thanks! Can you please provide a simple test for this? 
Otherwise, we can face regressions in the future! 😞

Reply:
Makes sense, added a regression test.